### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Shrirama-Upadhya-A**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined which repository your enhancement suggestion is related to, create an issue on that repository and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of project which the suggestion is related to.
```
Comment (amartya-dev) [inline CONTRIBUTING.md:22]: Please also add the point that the application has to use blocs for state management

**Review by amartya-dev — CHANGES_REQUESTED** [review]

**Review by amartya-dev — CHANGES_REQUESTED** [review]
This does not provide any files to look at, please either revert this commit or make sure the files are visible

**Review by amartya-dev — APPROVED** [review]
This looks good

**Review by amartya-dev — CHANGES_REQUESTED** [review]
You need to look into the last commit or revert it for me to merge

File: CONTRIBUTING.md (hunk 1 above)
Comment (Shrirama-Upadhya-A) [reply CONTRIBUTING.md:22]: Yeah ok

**Review by amartya-dev — APPROVED** [review]

**Pull request merged**